src/solution.h

```diff
@@ -35,7 +35,11 @@ typedef enum {
 
 #define MAX_AGE_OF_DIFFERENTIAL 1.0
 
-#define DGNSS_TIMEOUT S2ST(2)
+/*Number of solution periods before timing out*/
+#define DGNSS_PERIODS 2
+#define DGNSS_TIMEOUT_MSEC (DGNSS_PERIODS * 1/((float) soln_freq) * 1000)
```

Comment:
hmm is soln_freq the name of a variable?
would it be better to make that a macro parameter?